test/xpu_api/numerics/complex.number/complex.member.ops/divide_equal_complex.pass.cpp

```diff
@@ -29,11 +29,12 @@ test()
 
     dpl::complex<T> c3;
 
-    c3 = c;
-    dpl::complex<int> ic (1,1);
-    c3 /= ic;
-    assert(c3.real() ==  0.5);
-    assert(c3.imag() == -0.5);
+    // Commented to avoid compile error: no viable conversion from 'std::complex<int>' to 'const std::complex<long double>::_ComplexT' (aka 'const _Complex long double')
```

Comment:
Too many details and too many comments for the code to submit. Need to discuss a fix

Reply:
At any time ready to discuss. online or offline - up to you.